bokehjs/test/models/renderers/glyph_renderer.coffee

```diff
@@ -55,7 +55,7 @@ describe "GlyphRenderer", ->
       hit_test: (geometry) -> return create_1d_hit_test_result([])
 
     class HitTestHit
-      hit_test: (geometry) -> return create_1d_hit_test_result([0, 1])
+      hit_test: (geometry) -> return create_1d_hit_test_result([[0], [1]])
```

Comment:
``create_1d_hit_test_result`` accepts a list of lists. There's optionally a second element that the hit index is sorted by. The old behavior was returning an array of ``undefined`` values.